sympy/core/function.py

```diff
@@ -751,24 +751,16 @@ def fdiff(self, argindex=1):
         ix = argindex - 1
         A = self.args[ix]
         if A._diff_wrt:
-            if len(self.args) == 1:
+            if len(self.args) == 1 or not A.is_Symbol:
```

Comment:
What will happen when `A.is_Symbol` is `None`?

Reply:
Just go through @smichr comments, you will understand:https://github.com/sympy/sympy/issues/16140#issuecomment-469861158, https://github.com/sympy/sympy/issues/16140#issuecomment-469869532

Reply:
The `is_Symbol` attribute will only be `True` or `False`. It's basically a shorthand for `isinstance(obj, Symbol)`.